mRemoteV1/Resources/Help/ui_menus.htm

```diff
@@ -0,0 +1,224 @@
+    </tr>
+    <tr>
+      <td>Save Connection File As...</td>
+      <td>Saves the current connection file to a specific location on disk.</td>
+    </tr>
+    <tr>
+      <td>Delete...</td>
+      <td>Delete current selected item in connections dialog.</td>
+    </tr>
+    <tr>
+      <td>Rename</td>
+      <td>Rename current selected item in connections dialog.</td>
+    </tr>
+    <tr>
+      <td>Duplicate</td>
+      <td>Duplicate current selected item in connections dialog.</td>
+    </tr>
+    <tr>
+      <td>Reconnect All Open Connections</td>
+      <td>Sends a reconnect to all the open connections in mRmoteNG.</td>
+    </tr>
+    <tr>
+      <td>Exit</td>
+      <td>Exit mRemoteNG application</td>
+    </tr>
+  </table>
+  <!-- File menu - #end -->
+
+  <!-- View menu - #start -->
+  <h1 id="view_menu">View Menu</h1>
+  <hr>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td>Add Connection Panel</td>
+      <td>Create a new and empty panel.</td>
+    </tr>
+    <tr>
+      <td>Connection Panels</td>
+      <td>Jump to panel. Useful if you are organised with panels in mRemoteNG to sort out open connections.</td>
+    </tr>
+    <tr>
+      <td>Connections</td>
+      <td>Show connections dialog</td>
+    </tr>
+    <tr>
+      <td>Config</td>
+      <td>Show config dialog</td>
+    </tr>
+    <tr>
+      <td>Notifications</td>
+      <td>Show notifications dialog</td>
+    </tr>
+    <tr>
+      <td>Screenshots</td>
+      <td>Open Screenshots panel (See: <a href="ScreenshotManager.htm">ScreenshotManager</a> for more information)</td>
+    </tr>
+    <tr>
+      <td>Jump To</td>
+      <td>Place focus on "Connections and Config" or "Notifications" panel based on selection.</td>
+    </tr>
+    <tr>
+      <td>Reset layout</td>
+      <td>Resets the layout of panels and dialogs. Warning will come up about the action before continuing.</td>
+    </tr>
+    <tr>
+      <td>Lock toolbar positions</td>
+      <td>Locks the toolbars at the top of the application so you do not move around items by mistake.</td>
+    </tr>
+    <tr>
+      <td>Quick Connect Toolbar</td>
+      <td>Show quick connect toolbar</td>
+    </tr>
+    <tr>
+      <td>External Tools Toolbar</td>
+      <td>Show external tools toolbar</td>
+    </tr>
+    <tr>
+      <td>Multi SSH Toolbar</td>
+      <td>Show multi ssh toolbar</td>
+    </tr>
+    <tr>
+      <td>Fullscreen</td>
+      <td>Fullscreen mRemoteNG (will not fullscreen connection window but only the mRemoteNG application)</td>
+    </tr>
+  </table>
+  <!-- View Menu - #end -->
+
+  <!-- Tools Menu - #start -->
+  <h1 id="tools_menu">Tools Menu</h1>
+  <hr>
+  <table>
+    <tr>
+      <th>Item</th>
+      <th>Description</th>
+    </tr>
+    <tr>
+      <td>SSH File Transfer</td>
+      <td>Show SSH file transer panel (See: <a href="SSHFileTransfer.htm">SSH File Transfer</a> for more details)</td>
```

Comment:
These links are also broken for me when clicking them in mRemoteNG - maybe need to use `href="Help\SSHFileTransfer.html"`?

Reply:
Ah yes....I forgot to fix the file linking after rename of files. (to be more organized source wise. Fixed now